modules/rosa-hcp-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-hcp-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-upgrading-cli_{context}"]
+= Upgrading with the ROSA CLI
+
+You can upgrade a {hcp-title} (ROSA) cluster manually by using the ROSA CLI (`rosa`). This method schedules the cluster for an immediate upgrade, if a more recent version is available.
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA CLI on your installation host.
+
+.Procedure
+
+. To verify the current version of your cluster, enter the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name|cluster_id> <1>
+----
+<1> Replace `<cluster_name|cluster_id>` with the cluster name or the ID of the cluster.
+
+. To verify that a control plane upgrade is available, enter the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster=<cluster_name|cluster_id>
+----
++
+The command returns a list of versions to which the cluster can be upgraded, including a recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.8   recommended
+4.14.7
+4.14.6
+----
+
+. To verify that a machinepool upgrade is available, enter the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster <cluster-name> --machinepool <machinepool_name>
+----
++
+The command returns a list of the versions to which your machinepools can be upgraded, including a recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.5   recommended
+4.14.4
+4.14.3
+4.14.2
+4.14.1
+----
++
+[NOTE]
+====
+Your machinepools will not show versions beyond your cluster's current version.
+====
+
+. You can upgrade your cluster's control plane or individual machine pools on your cluster.
+** Run the following command to upgrade only the control plane of your {hcp-title} cluster.
++
+[source,terminal]
+----
+$ rosa upgrade cluster -c <cluster_name> --control-plane [--schedule-date=XX --schedule-time=XX] [--version 4.14.8]
+----
++
+Your control plane will be scheduled to be upgraded.
```

Comment:
Should this be changed to ' Your control plane is scheduled to be upgraded' - as the scheduling has already taken place, right, and it's the upgrade that will be occurring in the future? Also can it be modified to let users know when it will be upgraded- depending on the --schedule-date and --schedule-time variable perhaps?